amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/fanout/FanOutRecordsPublisher.java

```diff
@@ -25,7 +25,10 @@
 import org.reactivestreams.Subscriber;
 import org.reactivestreams.Subscription;
 
+import lombok.Data;
+import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
```

Comment:
Looks like these imports were added, but not used.

Reply:
Not sure why organize imports was ignored.